config/custom-environment-variables.yaml

```diff
@@ -0,0 +1,13 @@
+server:
+  hostname: 'HOSTNAME'
+  port: 'PORT'
+  adminPort: 'ADMIN_PORT'
+
+authentication:
+  username: 'USER'
+  password: 'PASSWD'
+
+database:
+  connectString: 'DBURL'
+  user: 'DBUSER'
+  password: 'DBPASSWD'
```

Comment:
Since this file exists and is named correctly by default, the environment variables will be enabled out of the box. Should this be the case? These variables will override any values in config/default.yaml which was confusing for me since `USER` was already set on my machine. Maybe these variables could be commented out by default or the file could be renamed so it can be copied to the correct filename or a note in the readme can be made that these will be active by default?

Reply:
Good point. @jaredkosanovic and I also talked about this issue before. To be honest, I'm not sure which one is the better solution. Let's discuss this as a group next week.

Reply:
I was gone a few days last week so I'm not sure if there has been any updates on this. @tsoliangwu0130 do you have any proposed solutions?

Reply:
After discussing with @jaredkosanovic, I agreed and made the change as @ianbrown9475 suggested :)